### PR TITLE
Adding toggle for unit change

### LINKED ISSUE #11: Unit changes
Allow users to use imperial vs metric units convert units either in frontend or backend

### PR DESCRIPTION
**Pull request by srina1h**
closes #11

### CODE REVIEW

File: frontend/src/components/Profile.js [hunk 1]
```diff
@@ -222,6 +239,18 @@ function Profile(props) {
...
+                  color="primary"
+                  value={units}
+                  exclusive
+                  onChange={handleUnitChange}
+                  aria-label="Platform"
+                >
+                  <ToggleButton value="imperial">Imperial</ToggleButton>
+                  <ToggleButton value="metric">Metric</ToggleButton>
+                </ToggleButtonGroup>
+              </Box>
```
Comment (shashank-madan) [inline frontend/src/components/Profile.js:253]: Can the label/title of the textinput boxes change to the respective units when toggled? (lbs to Kg, ft to m)

**Review by shashank-madan — CHANGES_REQUESTED** [review]
Add/toggle weight units (lb to kg, ft to m)

File: frontend/src/components/Profile.js (hunk 1 above)
Comment (srina1h) [reply frontend/src/components/Profile.js:259]: Nice idea

**Comment by srina1h** [comment]
Note that the unit change is kg-lbs and ft-cm

**Pull request merged**